rules/Php80/Rector/Class_/AnnotationToAttributeRector.php

```diff
@@ -369,4 +379,28 @@ private function collectDoctrineTagAndAnnotationToAttributes(
 
         return $doctrineTagAndAnnotationToAttributes;
     }
+
+    private function changeResolvedClassInContext(
```

Comment:
Please move this logic to separate service, eg under https://github.com/rectorphp/rector-src/tree/main/packages/BetterPhpDocParser/Attributes so we can have PHPStan CI green, thank you